### PR TITLE
transcoding: support path binding

### PR DESCRIPTION
**Pull request by lizan**

### CODE REVIEW

**Comment by lizan** [comment]
Not yet, I will write one in follow up PRs.

File: src/envoy/transcoding/config.cc [hunk 1]
```diff
@@ -82,49 +84,103 @@ Config::Config(const Json::Object& config, Server::Instance& server) {
...
+          break;
+        default:
+          break;
+      }
+
+      pmb.Register("POST", "/" + service->full_name() + "/" + method->name(),
+                   "", method_info);
+    }
+  }
+
```
Comment (qiwzhang) [inline src/envoy/transcoding/config.cc]: It is better to log a error if no methods added.  It is to detect config error, such as a typo in the services field or missing services field.

Comment (qiwzhang) [inline src/envoy/transcoding/config.cc]: or even throw an exception

Comment (lizan) [inline src/envoy/transcoding/config.cc]: throw an exception for services not in descriptor.

**Pull request merged**